src/main/resources/Message_de.properties

```diff
@@ -10,6 +10,7 @@ NO_TRANSIT_TIMES = Keine Fahrzeiten verfügbar. Das Datum kann zu weit in der Ve
 GEOCODE_FROM_NOT_FOUND    = Ausgangspunkt unbekannt. Können Sie bitte ein Wenig detailierter beschreiben?
 GEOCODE_TO_NOT_FOUND      = Fahrtziel unbekannt. Können Sie bitte ein Wenig detailierter beschreiben?
 GEOCODE_FROM_TO_NOT_FOUND = Ausgangspunkt und Fahrtziel unbekannt. Können Sie bitte ein Wenig detailierter beschreiben?
+GEOCODE_INTERMEDIATE_NOT_FOUND = An intermediate destination is unknown. Can you be a bit more descriptive?.

```

Comment:
I though the default value is used if it is missing here. Not, sure what is the best-practice:
 1. Only list english text in default properties file or...
 2. duplicate key and text (as here). This will make it easier for a "translator" to go in an edit the file.

Reply:
I would say duplicating the keys make it easier to see what is missing in a given language.